hw/dv/tools/dvsim/testplans/sec_cm_count_testplan.hjson

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: hardened_counter_check
+      desc: ''' Verify design behavior is correct when the harden counter is changed to an
+            unexpected value.
+
+            Stimulus:
+            - At the falling edge (non-active edge), force the counter to a different value.
+            - Randomly flip the value back to any other value to ensure the error is latched and
+              won't go away until reset.
+
+            Same checks as `one_hot_check`'''
```

Comment:
Perhaps copy-paste, since it is in a different file (which may or may not be imported)?
Also, put the closing `'''` on newline.

Reply:
make sense. Updated, thanks